app/board.js

```diff
@@ -11,8 +11,11 @@ const boardHandler = (req, res) => {
     headers: { 'User-Agent': config.user_agent }
   }
 
+  const limit = req.query.limit ? req.query.limit : 20;
+  const offset = req.query.offset ? req.query.offset : 0;
```

Comment:
Можно уменьшить дублирование кода такой записью:
```suggestion
  const limit = req.query.limit || 20;
  const offset = req.query.offset || 0;
```